src/main/java/proj/kedabra/billsnap/presentation/resources/AnswerNotificationResource.java

```diff
@@ -0,0 +1,17 @@
+package proj.kedabra.billsnap.presentation.resources;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.NotNull;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+@Data
+public class AnswerNotificationResource implements Serializable {
+
+    @NotNull
```

Comment:
if it's not null, why not just use the primitive boolean

Reply:
Error message with BindingResult is more descriptive with NonNull

Reply:
Furthermore, having primitive boolean causes bugs during endpoint call -> null is being read as false